getting-started.md

```diff
@@ -758,76 +758,73 @@ In this section, we will provide an overview of the supply chain security use ca
 Currently, this component supports cosign signatures and its key formats.
-It will work with open source cosign, kpack and Tanzu Build Service (which is what we will overview in this document).
+Although this component does not sign container images, you could use tools such
+as the cosign CLI, kpack, and Tanzu Build Service (which is what we will overview
+in this document) to generate signatures for your images.
 
-Signing an artifact creates metadata about it that allows consumers to verify its origin and integrity.
-By verifying signatures on artifacts prior to their deployment,
-this allows operators to increase their confidence that trusted software is running on their clusters.
+Signing an artifact creates metadata about it that allows consumers to verify
+its origin and integrity.
+Operators can increase their confidence that trusted software is running on their
+clusters by verifying signatures on artifacts prior to their deployment.
 
-**Use Cases**
+#### Use Cases
 
 * Validate signatures from a given registry.
+* Deny unsigned or unverified images from being admitted in the cluster.
+
+> **Note**: this component does not verify images that are already running in a
+> cluster.
 
 **Signing Container Images**
 
-Tanzu Application Platform supports verifying container image signatures that follow the `cosign` format.
-Application operators may apply image signatures and store them in the registry in one of several ways:
+Tanzu Application Platform supports verifying container image signatures that
+follow the cosign format.
+Application operators may sign container images and store them in the registry
+in several different ways, including:
 
 * Using Tanzu Build Service v1.3
-
 * Using [kpack](https://github.com/pivotal/kpack/blob/main/docs/image.md#cosign-config) v0.4.0
-
 * Signing existing images with [cosign](https://github.com/sigstore/cosign#quick-start)
 
+**Supplying secrets for registries protected by authentication**
 
-**Configure the Image Policy Webhook**
-
-After the webhook is running, create a service account named `image-policy-registry-credentials` in the `image-policy-system` namespace. This is required, even if the images and signatures are in public registries.
-
-After the image policy webhook is installed in the cluster, configure the image policy you want to enforce and the credentials to access private registries.
-
-**Configure a service account to hold private registry secrets**
+If your images and signatures are hosted in a registry protected by authentication
+you will need to provide the webhook with credentials to pull those signatures.
 
-When the platform operator is expecting to verify signatures stored in a private registry,
-it is required that you configure a service account with all the secrets for those private registries.
-The service account:
+If your resources already have `imagePullSecrets` configured, either directly
+in their manifests or via the `ServiceAccount` they run authenticated as,
```

Comment:
How could I achieve this scenario ?

```
via the `ServiceAccount` they run authenticated as,
```

Is it explained anywhere else ?

Reply:
We didn't add any instructions on how to do this because it is a Kubernetes-default style of auth. Do you think it would be worth linking to the Kubernetes Secret documentation where steps are explained on how to do so?

Reply:
Yes, i believe linking to the kubernetes documentation is really good in case it isn´t known